### ISSUE TITLE
Idea - Maintain an itemized list of "Currently Recommended" hardware components that someone (developer/anyone) can purchase to start building a station

### ISSUE THREAD

**Issue by danlohman** [Sky360-Repository/sky360#19]
We have the list of hardware components on the website here: https://www.sky360.org/curated-list-of-hardware
and we have the list of components listed in the [how to build a station document](https://cdn.website-editor.net/s/905d39c38a3742b8900eeb2395eaa83c/files/uploaded/How%2520to%2520build%2520a%2520station%2520v1.6.pdf?Expires=1689693115&Signature=Y2ZlLM-RbS-JmmJkfF3cN97J3x8iv4DsM4pelJzQyrRKM~eyWKg3d4nvz04MdZd7af4kJr5lIlzrz0mBIE5hqNjV8ogXUflaWUYtfzwYR9G16on~4~e8YLEibh-wSfbp59~-H6hkEHPnp5yPZDd7wTT6nRwdLAq5EOXVbQa3MuKvB9slURMvGfIC41tJ5t6RdCsa6hypLG320cwZfIycKxk8zFPb7h8yYwGvKhbZ7BkZmrXw7bDFOTmHZcLYiPpUZOdIjUOqU~E5uACESO~D1rV4nGpI3517PpuZSXsKzkN88qHtkhcvbXD~lD0PEy~qpXGZQOpg4EcICuAEdraFdQ__&Key-Pair-Id=K2NXBXLF010TJW)
We don't have a itemized list which scopes the quantity per part of each component that is designed/specified/recommended.
We should make it very easy for someone who wants to get caught up with the current progress of the station design/build to get the list of recommended parts and source/buy/find/print those parts that have been specified.
I have started this google sheet list of parts that need review: https://docs.google.com/spreadsheets/d/1oK2koXBDekzeqpIH9e6gXaRhBRWGVhKSeIW8s-T7TQ8/edit#gid=0

**Comment by realsnick** [comment]
@RichardGHopf can you help answering?

**Comment by Sky360Admin** [comment]
if this is a kind of checklist, i would expect direct links for procurement.
shall this be part of the how-to document?

**Comment by danlohman** [comment]
I agree that procurement links would be great to include and in addition we could integrate it into the how to documents as well.  I first want to make sure that the list I compiled from the website and how-to build document is accurate and up to date.  For example, I think that the list of components on the website contains a few items that may be alternative options, in which case only one of the alternatives should be purchased.   Having not actually built a station myself I didn't want to assume.  If someone can proof read and correct any errors on the list, I can take care of adding links, supplemental info, data sheets, ext.  Thanks!
